src/ducks/myApps/Container.jsx

```diff
@@ -1,6 +1,6 @@
 import { connect } from 'react-redux'
 
-import { fetchApps, uninstallApp } from './index'
+import { fetchMyApps, uninstallApp } from './index'
```

Comment:
I hate the `My` prefix for variable or Classes. I think it is too confusing. What about `InstalledApps` here ?

Reply:
Completely agree 👍